manifests/qpid.pp

```diff
@@ -18,12 +18,24 @@
     require   => Service['qpidd'],
     logoutput => true,
   } ~>
-  exec { 'bind katello entitlements queue to qpid exchange messages that deal with entitlements':
-    command   => "qpid-config --ssl-certificate ${client_cert} --ssl-key ${client_key} -b 'amqps://localhost:5671' bind event ${candlepin_event_queue} '*.*'",
-    onlyif    => "qpid-config --ssl-certificate ${client_cert} --ssl-key ${client_key} -b 'amqps://localhost:5671' queues ${candlepin_event_queue}",
-    path      => '/usr/bin',
-    require   => Service['qpidd'],
-    logoutput => true,
+  qpid::bind_event { 'entitlement.created':
```

Comment:
@ekohl After talking to @pmoravec with passing the striped client crt like here, https://github.com/Katello/katello-packaging/blob/master/katello/katello-debug.sh#L81 we dont need the key.